README.md

```diff
@@ -119,5 +119,5 @@ The SDK comes with some unit tests, to run them, execute the following command:
 
 ### <a name="restapidoc"></a>Rest API documentation
 
-Detailed documentation of the Server Rest API can be found on your local Dynatrace Server: [https://DTSERVER:8021/api-docs/current/](https://localhost:8021/api-docs/current/)
+[Server Rest interfaces wiki](https://www.dynatrace.com/support/doc/appmon/appmon-reference/rest-interfaces/server-rest-interfaces/)
```

Comment:
move to #Services section, change 'wiki' to 'documentation', wrap it around a meaningful sentence